app.py

```diff
@@ -1724,7 +1761,8 @@ def export_boundary():
     location = request.args.get("location")
     export_format = request.args.get("format")
 
-    boundary_data, _ = generate_geojson_osm(json.loads(location), streets_only=False)
+    boundary_data, _ = generate_geojson_osm(
+        json.loads(location), streets_only=False)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider handling error return value</b></div>
<div id="fix">

Consider improving code readability by using a more descriptive variable name for the second return value from `generate_geojson_osm()` since it's being discarded. The underscore convention suggests it's unused but it actually contains error information.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
    boundary_data, error = generate_geojson_osm(
        json.loads(location), streets_only=False)
    if error:
        return jsonify({'error': error}), 400
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/realronaldrump/everystreet-new/pull/95#issuecomment-2543418263>#d8630f</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged